introduction.md

```diff
@@ -0,0 +1 @@
+Hello - my name is Eric Lamy. I'm a developer and project manager in Newburyport, MA.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.